packages/cli/src/commands/init/init.js

```diff
@@ -14,8 +14,10 @@ import {
   executePostInitScript,
 } from './template';
 import {changePlaceholderInTemplate} from './editTemplate';
-import * as PackageManager from '../../tools/PackageManager';
+import * as PackageManager from '../../tools/packageManager';
 import {processTemplateName} from './templateName';
+import banner from './banner';
+import getLoader from '../../tools/loader';
```

Comment:
can we stick to the module name? So `loader` instead of `getLoader`

Reply:
I'll just replace default export. I think `getLoader` is good naming here. It shouldn't be confusing when I use `import { getLoader } from '../../tools/loader`;